components/Player/PlayerMMR.jsx

```diff
@@ -4,16 +4,44 @@ import Spinner from '../Spinner';
 import { connect } from 'react-redux';
 import { player } from '../../reducers';
 import styles from './PlayerHeader.css';
+import SocialPerson from 'material-ui/svg-icons/social/person';
+import SocialGroup from 'material-ui/svg-icons/social/group';
+import ActionHelp from 'material-ui/svg-icons/action/help';
+import { blue500, orange500, grey500 } from 'material-ui/styles/colors';
 
 export const PlayerMMR = ({ loading, error, rank, soloRank, mmrEstimate }) => {
   const getPlayerMMR = () => {
     if (error) return <Error />;
     if (loading) return <Spinner />;
     return (
-      <div className={styles.mmr}>
-        <div>Solo: {soloRank}</div>
-        <div>Party: {rank}</div>
-        <div>Estimate: {mmrEstimate.estimate}</div>
+      <div>
+        <span>
+          <span>
+            <abbr title={"Solo MMR"}>
+              <SocialPerson color={blue500} />
+            </abbr>
+          </span>
+          <small>{soloRank}</small>
+        </span>
+        <span>
+          <span>
+            <abbr title={"Party MMR"}>
```

Comment:
Feels like there are too many spans in a row in here. I haven't looked closely but you might not need this many tags nested.